tests/functional/dummy-skeleton/__packagename__/__init__.py

```diff
@@ -0,0 +1,2 @@
+
+VERSION = '1.0.0'
```

Comment:
Should we have an option to somehow update the `dummy-skeleton` automatically when the `skeleton` is changed?

Reply:
I believe this `dummy-skeleton` will change over time to adapt to our testing needs, diverging from an empty project `skeleton`. Therefore I don't thing auto-updating it would be a good idea.